Formula/syslog-ng.rb

```diff
@@ -0,0 +1,50 @@
+class SyslogNg < Formula
+  desc "Log daemon with advanced processing pipeline and a wide range of I/O methods"
+  homepage "https://www.syslog-ng.com"
+  url "https://github.com/syslog-ng/syslog-ng/releases/download/syslog-ng-3.38.1/syslog-ng-3.38.1.tar.gz"
+  sha256 "5491f686d0b829b69b2e0fc0d66a62f51991aafaee005475bfa38fab399441f7"
+  license all_of: ["LGPL-2.1-or-later", "GPL-2.0-or-later"]
+
+  depends_on "autoconf" => :build
+  depends_on "autoconf-archive" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "glib"
+  depends_on "hiredis"
+  depends_on "ivykis"
+  depends_on "json-c"
+  depends_on "libdbi"
+  depends_on "libmaxminddb"
+  depends_on "libnet"
+  depends_on "librdkafka"
+  depends_on "mongo-c-driver"
+  depends_on "openssl@1.1"
+  depends_on "pcre"
+  depends_on "python@3.11"
+  depends_on "riemann-client"
+
+  uses_from_macos "curl"
+
+  def install
+    sng_python_ver = Formula["python@3.11"].version.major_minor
+
+    system "./configure", *std_configure_args,
+                          "--disable-silent-rules",
+                          "--sbindir=#{bin}",
```

Comment:
Why not use the real sbin here?

Reply:
My understanding is that a Homebrew-installed syslog-ng instance does not require root privileges to be used properly (unless one configures a network source using a low port number).

The idea is coming from `Formula/httpd.rb` and `Formula/lighttpd.rb`.

Reply:
Nothing in Homebrew requires root so sbin is safe to use. I think those were fixes for another problem with dependencies.

Reply:
I see. Thank you. I'm removing the sbin override.